### PR TITLE
chore(infra): Gen 1 Refresh

### PR DESCRIPTION
**Pull request by Equartey**
*Description of changes:*
- Updates Infra stack to achieve a clean deploy
- Removes old infra stacks (analytics & storage) that are now tested in Gen 2
- Fixed some type issues
- Lots of auto formatting changes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by khatruong2009 — APPROVED** [review]

File: infra/lib/auth/stack.ts [hunk 1]
```diff
@@ -260,7 +260,6 @@ class AuthIntegrationTestStackEnvironment extends IntegrationTestStackEnvironmen
     const customSenderKmsKey = new kms.Key(this, "CustomSenderKey", {
       description: `Key for encrypting/decrypting SMS messages sent from ${this.name} user pool`,
       removalPolicy: RemovalPolicy.DESTROY,
-      alias: this.name,
```
Comment (NikaHsn) [inline infra/lib/auth/stack.ts:263]: Q: don't we need the alias anymore?

File: infra/lib/common.ts [hunk 2]
```diff
@@ -14,14 +14,20 @@ import * as secretsmanager from 'aws-cdk-lib/aws-secretsmanager';
...
   account: process.env.CDK_DEFAULT_ACCOUNT,
   region: 'us-west-2',
 };
 
+export enum StorageAccessLevel {
+  public = "guest",
+  protected = "protected",
+  private = "private",
+}
+
```
Comment (NikaHsn) [inline infra/lib/common.ts:30]: I'm not sure if we need this anymore, we deleted the storage stack and hence we can remove storage config and also not adding it to the AmplifyConfig. we can also remove the analytic config and not add it to AmplifyConfig

File: infra/lib/auth/stack.ts (hunk 1 above)
Comment (Equartey) [reply infra/lib/auth/stack.ts:263]: This was causing my deployment to fail. Unsure what the cause was, but don't see the value in having defined. Also when this lifted to the gen 2 stack, it was dropped.

File: infra/lib/common.ts (hunk 2 above)
Comment (Equartey) [reply infra/lib/common.ts]: Good call out, I pushed a update removing the extra code.

**Review by khatruong2009 — APPROVED** [review]

**Review by NikaHsn — APPROVED** [review]

**Pull request merged**